File: Tests/Resources/ExpectedTypeScriptCode/Inputs/MailingAddressInput.ts
import { ID } from "../GraphApi"
import {
  CountryCode,
} from "../Enums"

export interface MailingAddressInput {
  address1?: string | null;
  address2?: string | null;
  city?: string | null;
  company?: string | null;
  country?: string | null;
  countryCode?: CountryCode | null;
  firstName?: string | null;
  id?: ID | null;
  lastName?: string | null;
  phone?: string | null;
  province?: string | null;
  provinceCode?: string | null;
  zip?: string | null;
}

File: Tests/Resources/ExpectedKotlinCode/GenerateMultipleFiles/Inputs/MailingAddressInput.kt

package com.shopify.syrup.inputs

import com.shopify.foundation.syrupsupport.*
import com.shopify.syrup.enums.*
import java.math.BigDecimal
import org.joda.time.DateTime;
import javax.annotation.Generated

@Generated("com.shopify.syrup")
data class MailingAddressInput(var address1: InputWrapper<String?> = InputWrapper(), var address2: InputWrapper<String?> = InputWrapper(), var city: InputWrapper<String?> = InputWrapper(), var company: InputWrapper<String?> = InputWrapper(), var country: InputWrapper<String?> = InputWrapper(), var countryCode: InputWrapper<CountryCode?> = InputWrapper(), var firstName: InputWrapper<String?> = InputWrapper(), var id: InputWrapper<ID?> = InputWrapper(), var lastName: InputWrapper<String?> = InputWrapper(), var phone: InputWrapper<String?> = InputWrapper(), var province: InputWrapper<String?> = InputWrapper(), var provinceCode: InputWrapper<String?> = InputWrapper(), var zip: InputWrapper<String?> = InputWrapper())
